client/src/components/Mylist/MyAnimeList.js

```diff
@@ -1,18 +1,44 @@
-import React from 'react'
-
+import React, { useState } from 'react'
+import { useAuth0 } from '@auth0/auth0-react'
 const MyAnimeList = () => {
-
+const {user} = useAuth0();
 /* make a get request for items in saved list
 fetch url ${user.sub}
 setLts(list)
 
 useEffect for authentication
 */
 
+const [list, setList] =useState([])
```

Comment:
Instead of calling this route ill need to call the rote for my anime table that is a joined query based on mal_id associated to user

This route will be created in the animeList.js